execution_workflows/README.md

```diff
@@ -76,6 +76,32 @@ For the codes please refer to the following documents:
  `AA/MISO/AA_MISO_01.bed` would be the output of MISO (your method) for the identification challenge (OUTCODE 01, we know that from [`execution_output_specification.md`][outcode]), run on dataset "P19" using 4 cores (PARAMCODE AA, we know that from) [`summary_input_specification.md`][param-code])
 
 
+## Tools
+List of tools used in APAeval. Please update columns as the execution workflows progress.
+
+| Method | Citation | Repository | Type | Status in APAeval | Present in OpenEBench | Installation | Can take your dog for a walk? |
```

Comment:
Also, I've thought a little bit about "Installation", and I think that's not what's relevant to most people as it would rarely have anyone change their mind about using a tool. I think what would be better is to have a column "Packaging" with one or more of the following options:
- `Conda: [channels]` (list `bioconda`, if available; if available but a version from another channel was downloaded more often, list that one, too)
- `Containers: [maintainers]` (list one or both of `BioContainers` and author's `official` repository)
- `Workflow: [workflow language]` (list one of `CWL`, `Galaxy`, `Nextflow`, `Snakemake`, `WDL` or `other`)
- `Other`, optionally with a package repository:
  - `PyPI`
  - `CRAN`
  - `Bioconductor`
  - `GitHub` - but only if it uses [GitHub _Packages_](https://github.com/features/packages) - it's a specific tab that only shows up if repos support it, which is not (yet) very commonly used - not just if the code is on GitHub any old way!
  - maybe there are others, but most likely not

Examples:

* `Conda: bioconda; Containers: BioContainers, official; Other: PyPI`
* `Workflow: Snakemake`
* `Workflow: other`
* `Other: Bioconductor`
* `Other`

Each channel, registry or repository should be linked.

As all of this is a lot of work, I think it's best to create an issue for that and first merge this version

Reply:
will put as new issue